esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +215,43 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    @unittest.skipIf(list(map(lambda x: os.getenv('OS_FUNCTIONAL_NODE_%s' % x),
+                              ['NAME', 'UUID'])) == [None, None],
+                     'Neither env variable OS_FUNCTIONAL_NODE_UUID \
+                      or OS_FUNCTIONAL_NODE_NAME is set')
+    def test_node_list_long(self):
+        """ Tests functionality "esi node list" using node_uuid or node name.
+            checks node_uuid or node name is present in node list or not.
+            Test steps:
+            1) Set either of the environment variable using
+               export OS_FUNCTIONAL_NODE_UUID = node_uuid or
+               export OS_FUNCTIONAL_NODE_NAME = node_name
+            2) It check that the output of "node list" contains
+               the node uuid or node name it's tested with. """
+        node_uuid = os.getenv('OS_FUNCTIONAL_NODE_UUID')
+        node_name = os.getenv('OS_FUNCTIONAL_NODE_NAME')
+        listings = esi.node_list(self.clients['admin'], long=True)
+        self.assertNotEqual(listings, [])
+        if node_uuid is not None:
+            self.assertIn(node_uuid, [x['UUID'] for x in listings])
+        if node_name is not None:
+            self.assertIn(node_name, [x['Name'] for x in listings])
+
+    @unittest.skipIf('OS_FUNCTIONAL_NODE_NAME' not in os.environ.keys(),
+                     'Environment variable OS_FUNCTIONAL_NODE_NAME not set')
+    def test_node_list_basic(self):
+        """ Tests basic functionality of "esi node list" it by default
+            checks if node_name is present in node list or not.
```

Comment:
This sentence is unbroken, which makes it somewhat hard to read. It's also unclear what "by default" means here since this test only does one thing-- I'm assuming you're referring to the fact that this test validates the default behavior of the "esi node list" command? I'd probably rewrite this to something along the lines of:

`Tests basic functionality of "esi node list" when default behavior is invoked. Checks if node_name is present in node list.`